docs/package.json

```diff
@@ -25,7 +25,7 @@
   "dependencies": {
     "@docusaurus/core": "2.4.1",
     "@docusaurus/preset-classic": "2.4.1",
-    "@stellar/freighter-api": "^2.0.0",
+    "@stellar/freighter-api": "latest",
```

Comment:
apparently if you use `latest`, yarn will ignore the workspace and just grab from the registry :tada